### PR TITLE
Create a new organization with just the mandatory user inputs

### PR DESCRIPTION
**Pull request by anusuyar**
VCDA-171(pyvcloud) - Create a new organization with just the mandatory user inputs

### CODE REVIEW

File: pyvcloud/vcd/system.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# VMware vCloud Director Python SDK
+# Copyright (c) 2014 VMware, Inc. All Rights Reserved.
```
[the remark this answers is not in the document]
Comment (anusuyar) [reply pyvcloud/vcd/system.py]: Changed.

File: pyvcloud/vcd/system.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+         Create new organization
+        :param org_name: The name of the organization
+        :param full_org_name: The fullname of the organization
+        :return: (AdminOrgType) Created org object
+         """  # NOQA
+        if self.admin_resource is None:
+            self.admin_resource = self.client.get_resource(self.admin_href)
+        org_params = E.AdminOrg(
+            E.FullName(full_org_name),
+            E.Settings,
```
Comment (rocknes) [inline pyvcloud/vcd/system.py]: Is an empty Settings tag necessary?

Comment (anusuyar) [reply pyvcloud/vcd/system.py]: Yes. the call fails without the settings tag.

File: tests/vcd_system.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+from pyvcloud.vcd.system import System
+from pyvcloud.vcd.test import TestCase
+
+
+class TestSystem(TestCase):
+    def test_create_org(self):
+        sys_admin = self.client.get_admin()
+        system = System(self.client, admin_resource=sys_admin)
+        org_name = "orgName".join(
```
Comment (rocknes) [inline tests/vcd_system.py]: move these static strings to the config file.

File: pyvcloud/vcd/system.py (hunk 2 above)
Comment (sompa) [reply pyvcloud/vcd/system.py]: Would be it be good to spit out a message that the following org<org_name> has been created ?

File: tests/vcd_system.py (hunk 3 above)
Comment (anusuyar) [reply tests/vcd_system.py]: I didn't want org_name getting confused with org in config.yml

File: pyvcloud/vcd/system.py (hunk 2 above)
Comment (anusuyar) [reply pyvcloud/vcd/system.py]: I am printing the 'created message in ' vcd-cli.

**Pull request merged**